src/main/java/de/retest/recheck/ignore/SearchFilterFiles.java

```diff
@@ -35,12 +35,12 @@
 
 	private SearchFilterFiles() {}
 
-	public static List<Pair<Path, FilterLoader>> getAllFilterFiles() {
+	public static List<Pair<String, FilterLoader>> getAllFilterFiles() {
```

Comment:
Wasn't it your suggestion to use `Path` instead of `String` to reflect that filters don't have a name but only a path? Moreover, `Path#getFileName()` only returns the filename, so doesn't this cause duplicates if the user creates a filter with the same name as a predefined one?